<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceScreen</class>
 <widget class="QDialog" name="DeviceScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 13, 32);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>372</width>
     <height>255</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(29, 13, 32);
color: rgb(73, 236, 253);</string>
          </property>
          <property name="text">
           <string>Выбранные места:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QListWidget" name="devicesListWidget">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>181</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>181</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(29, 13, 32);
color: rgb(73, 236, 253);
border: 1px solid rgb(73, 236, 253);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="deviceButton_2">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;
border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="deviceButton_1">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;
border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="deviceButton_3">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;
border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="deviceButton_8">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="deviceButton_9">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="deviceButton_4">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="deviceButton_10">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="deviceButton_5">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;
border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="deviceButton_7">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="deviceButton_6">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;border: 1px solid rgb(131, 143, 143);</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(73, 236, 253);
color: rgb(29, 13, 32);
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
